test/integration/application-log-assertion-with-cucumber-report/webapp/test/WebsiteCode.js

```diff
@@ -0,0 +1,43 @@
+sap.ui.getCore().attachInit(function() {
+	"use strict";
+
+	const sText = "Number of lemmings saved: ";
+	let iNumSavedLemmings = 0;
+
+	const oTitle = new sap.m.Label({
```

Comment:
Currently only the sap.ui.core library is available in this integration test setup (see @openui5/sap.ui.core in package.json). I know that this is an example, but to keep this test minimal, I would prefer to not rely on other libraries.

This is why the tests are failing right now but not when using UI5 from the CDN.

Reply:
Wow, thanks, that was fast.
Let me see what I can come up with the core components.